README.adoc

```diff
@@ -1,63 +1,73 @@
-// Copyright (c) 2018 IBM Corporation and others.
+// Copyright (c) 2019 IBM Corporation and others.
```

Comment:
Generally for copyrights we keep the initial year something was produced and then append on the last year it had a significant modification, like: `Copyright (c) 2018, 2019 IBM Corporation and others.`.  Not sure if you're doing something different since the guide has not yet been released.

Reply:
Yes, this is new content as a guide. If we modified on a file that was previously published, we follow the convention you mentioned as well.